.github/workflows/deploy_mkdocs.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish docs via GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      # Only rebuild website when docs have changed
```

Comment:
Just a note that when you also display API docs, it might be worthwhile to rebuild the website more often.